gradle/libs.versions.toml

```diff
@@ -149,6 +151,8 @@ sun-activation = { group = "com.sun.activation", name = "javax.activation", vers
 
 selenium = { group = "org.seleniumhq.selenium", name = "selenium-java", version.ref = "selenium" }
 rauschig = { group = "org.rauschig", name = "jarchivelib", version.ref = "rauschig" }
+mybatis = { group = "org.mybatis", name = "mybatis", version.ref = "mybatis"}
+h2db = { group = "com.h2database", name = "h2", version.ref = "h2db"}
```

Comment:
What license does `h2` and `mybatis` use?

Reply:
`MyBatis` use the `The Apache Software License, Version 2.0` license, check it out here: https://mybatis.org/mybatis-3/licenses.html.  
H2 Database use `Mozilla Public License Version 2.0` and `Eclipse Public License - Version 1.0` license, check it out here: https://www.h2database.com/html/license.html

Reply:
cc @justinmclean Is ok to use H2 for us?

Reply:
Both the MPL and EPL are weak copyleft / Category B licenses. Our source can't include their code, but we could use them in a binary release.

Reply:
@justinmclean I'm not very familiar with this area. Based on your description, I understand that we can include their release versions as libraries into our project, rather than directly using their source code, is this correct?

Reply:
> @justinmclean I'm not very familiar with this area. Based on your description, I understand that we can include their release versions as libraries into our project, rather than directly using their source code, is this correct?

Yes.

Reply:
@xloya You should modify the `Licence.bin`, too.